acceptance/cmd/workspace/apps/run-local/script

```diff
@@ -0,0 +1,31 @@
+cd app
+
+# We first run the command with different entry poiny which starts unblocking script
+# so we don't need to start it in background. It will install the dependencies as part of the command
+trace $CLI apps run-local --prepare-environment --entry-point test.yml 2>&1 | grep -w "Hello, world"
+
+title "Starting the app in background..."
+$CLI apps run-local --prepare-environment --debug > ../out.run.txt 2>&1 &
+PID=$!
+
+cd ..
+
+title Waiting for the app to start...
+# Use a loop to check for the startup message instead of tail/sed which can be unreliable on Windows
+# due to file locking, buffering issues, and different text processing behavior across Windows versions.
+# A simple grep loop is more robust across platforms.
+while [ -z "$(grep -o "Python Flask app has started with" out.run.txt 2>/dev/null)" ]; do
+    sleep 1
+done
+
+title "Checking app is running..."
+trace curl -s -o - http://127.0.0.1:8001
+
+title "Sending shutdown request..."
+trace curl -s -o /dev/null  http://127.0.0.1:8001/shutdown || true
+
+title "Checking CLI command output..."
+
+trace grep "To debug your app, attach a debugger to port 5678" ./out.run.txt
```

Comment:
Is there a reasonably simple way to test if the debug server is up and running? Maybe some test network call we can make?

Reply:
Unfortunately making it cross platform and reliable is a bit difficult sue to nature of this endpoint as using simple curl call is not feasible